.github/workflows/ci.yml

```diff
@@ -240,14 +240,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - test_name: direct-perf
-            setup: echo 'Noop'
-          - test_name: relayed-perf
-            setup: |
-              #  Disallow traffic between gateway and client container
-              sudo iptables -I FORWARD 1 -s 172.28.0.100 -d 172.28.0.105 -j DROP
-              sudo iptables -I FORWARD 1 -s 172.28.0.105 -d 172.28.0.100 -j DROP
+        dim1: [1, 2, 3, 4, 5]
+        dim2: ['a', 'b', 'c', 'd', 'e']
+        test_name:
```

Comment:
This work!

Reply:
This could also be an array (see other tests).

yaml style checker needed? 😁

Reply:
Yeah it's working now that I've refactored things a little bit ;-).

We now run each performance test on its own.